local/modules/HookAnalytics/Form/Configuration.php

```diff
@@ -27,7 +27,9 @@ protected function buildForm()
     {
         $form = $this->formBuilder;
 
-        $value = ConfigQuery::read("hookanalytics_trackingcode", "");
+        $lang = $this->getRequest()->getSession()->get("thelia.admin.edition.lang");
+
+        $value = HookAnalytics::getConfigValue("hookanalytics_trackingcode", "", $lang->getLocale());
```

Comment:
Please can you verify that the lang is not null before calling his method ? 
And if null maybe take the default lang.